### PR TITLE
Fix the link to the automate requirements

### PR DESCRIPTION
**Pull request by tas50**
Bad link

### CODE REVIEW

**Review by susanev — APPROVED** [review]

**Comment by srenatus** [comment]
DCO please 😉

**Comment by mjingle** [comment]
@tas50 This does need a DCO. :)

**Comment by mjingle** [comment]
@tas50 Also, this is missing a colon at the end of the line. If you could add that, it would be greatly appreciated!

File: chef_master/source/platforms.rst [hunk 1]
```diff
@@ -225,7 +225,7 @@ Chef Automate Server
 Commercial Support
 ++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Commercial support for the `Chef Automate 2 Server <https://automate.chef.io/docs/system_requirements>`_ is available for platforms that use
+Commercial support for the `Chef Automate 2 Server <https://automate.chef.io/docs/system-requirements/>`_ is available for platforms that use
```
Comment (mjingle) [inline chef_master/source/platforms.rst]:
```suggestion
Commercial support for the `Chef Automate 2 Server <https://automate.chef.io/docs/system-requirements/>`_ is available for platforms that use:
```

**Review by mjingle — APPROVED** [review]
tenor-225176476

**Pull request merged**